docs/python/binary-extension.rst

```diff
@@ -259,12 +259,13 @@ The ``bezier._speedup`` module (``.pyd`` file) depends on this local copy of
 
 .. testsetup:: windows-libs, windows-extension, windows-dll
 
-   import distutils.ccompiler
    import os
    import pathlib
    import re
    import subprocess
 
+   import setuptools._distutils.ccompiler
```

Comment:
Relying on `setuptools._distutils` isn't my favorite, but figuring out the "right way" to do this isn't worth the time right now (and I don't have the time for it unfortunately).